File: 0023-FindWordsContainingCharacter/2942-FindWordsContainingCharacter.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> result = new ArrayList<>();
        for(int i = 0 ; i < words.length; i++){
            String xStr = Character.toString(x);
            if(words[i].contains(xStr))
                result.add(i);
        }

        return result;

    }
}

File: 0023-findWordsContainingCharacter/2942-findWordsContainingCharacter.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {

    let arrayOfIndices = [];

    for(let i = 0 ; i < words.length ; i++) {
        if (words[i].includes(x)) {
            arrayOfIndices.push(i);
        }
    }
    return arrayOfIndices;
};